net/dns/nm.go

```diff
@@ -202,6 +202,11 @@ func (m nmManager) SetDNS(config OSConfig) error {
 
 func (m nmManager) SupportsSplitDNS() bool { return false }
 
+func (m nmManager) GetBaseConfig() (OSConfig, error) {
+	// TODO
+	return OSConfig{}, nil
```

Comment:
return that `ErrGetBaseConfigNotSupported` error and let caller map `ErrGetBaseConfigNotSupported` to nil with a warning logged?

Reply:
that also gives you a good way to grep for these later, like why we added content.TODO() ... something that "works" right away but isn't quite right but is easy to find.

Reply:
If base config fetching doesn't work but the OS also doesn't support split DNS, split DNS configs will be actively broken, so unsure if NotSupported is the right knob for this. But I'll do that for now and see how it evolves.